dashboard/lib/sqs/queue_processor_config.rb

```diff
@@ -31,9 +31,40 @@ def initialize(queue_url:,
       @max_rate_proc = max_rate_proc
       @num_processors = check_not_nil(num_processors)
       @num_workers_per_processor = check_not_nil(num_workers_per_processor)
+      @name = name
       @logger = check_not_nil(logger)
     end
 
+    # Creates a config from an options hash.
+    def self.create(options)
+      SQS::QueueProcessorConfig.new(
+          queue_url: options['queue_url'],
+          handler: options['handler_class'].constantize.new,
+          num_processors: options['num_processors'] || 1,
+          num_workers_per_processor: options['num_workers_per_processor'] || 10,
+          initial_max_rate: options['initial_max_rate'] || 5000,
+          max_rate_proc: options['max_rate_proc'],
+          name: options['name'])
+    end
+
+    def self.create_configs_from_json(json)
+      processors_options = JSON::parse(json)['queues']
+      processors_options.map do |processor_option|
```

Comment:
i think this block parameter should be `|processor_options|` or more simply `options` to clarify it's still a hash of multiple entries and not just one